weavetest/abci.go

```diff
@@ -233,14 +230,10 @@ func (w *WeaveRunner) Get(key []byte) []byte {
 	if query.Code != 0 {
 		panic(query.Log)
 	}
-	// TODO: avoid importing app
-	var value app.ResultSet
-	err := value.Unmarshal(query.Value)
-	if err != nil {
-		// oh, for an error return here...
-		panic(errors.Wrap(err, "cannot parse values"))
+	var value ResultSet
```

Comment:
Cut and paste resultset is suboptimal. But yeah, good enough to merge.

We should really tackle this properly, as outlined in https://github.com/iov-one/weave/issues/412